packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialogDownloadFile/ExportWhiteboardDialogDownloadFile.test.tsx

```diff
@@ -71,11 +80,11 @@ describe('<ExportWhiteboardDialogDownloadFile />', () => {
       );
     };
 
-    jest.mocked(URL.createObjectURL).mockReturnValue('blob:url');
+    vi.spyOn(URL, 'createObjectURL').mockReturnValue('blob:url');
   });
 
   afterEach(() => {
-    jest.mocked(document.createElement).mockRestore();
+    vi.spyOn(document, 'createElement').mockRestore();
```

Comment:
Doesn't this create a new spy and then restores it again? The previous spy is then still there.
It should use the existing spy instead.